client/sites-dashboard/components/sites-sorting-dropdown.tsx

```diff
@@ -65,13 +65,13 @@ export const SitesSortingDropdown = ( {
 
 		switch ( sitesSorting ) {
 			case `lastInteractedWith${ SEPARATOR }desc`:
-				return __( 'Sorted automagically' );
+				return __( 'Sorting by: Automagically' );
```

Comment:
```suggestion
				return __( 'Sorting by: Magic' );
```

I _think_ this is more correct.

Reply:
Or **Automaggic** 🤣